.yardopts

```diff
@@ -1,3 +1,4 @@
 --no-private
 --embed-mixins
 --markup markdown
+--exclude lib/job-iteration/integrations/.*
```

Comment:
These are regexps not globs:
https://github.com/lsegal/yard/blob/v0.9.27/lib/yard/cli/yardoc.rb#L576
https://github.com/lsegal/yard/blob/v0.9.27/lib/yard/parser/source_parser.rb#L105